scripts/install.sh

```diff
@@ -670,6 +670,7 @@ function uninstall() {
         fi
     fi
 
+    echo "Uninstallation completed"
     exit 0
```

Comment:
I think this exit is redundant. We exit after returning from this function.

Reply:
Agree, removed